src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -140,7 +142,28 @@ export const ImportBlueprintModal: React.FunctionComponent<
     <Modal
       variant={ModalVariant.medium}
       isOpen={isOpen}
-      title={'Import pipeline'}
+      title={
+        <>
+          Import pipeline
+          <Popover
+            bodyContent={
+              <div>
+                You can import the blueprints you created by using the Red Hat
+                image builder into Insights images to create customized images.
+              </div>
+            }
+          >
+            <Button
+              variant="plain"
+              aria-label="About regions"
+              className="pf-v5-u-pl-sm header-button"
```

Comment:
The `header-button` class updates right padding only, might not be needed here.

```suggestion
              className="pf-v5-u-pl-sm"
```